evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -71,9 +72,10 @@ contract FluxAggregator is AggregatorInterface, Owned {
   uint8 public override decimals;
   bytes32 public description;
 
+  uint256 constant RESERVE_ROUNDS = 2;
```

Comment:
I am not familiar with the details of this code, but as an outsider... this contract seems to be becoming more and more complex.

Sometimes complexity is unavoidable, but I tend to be more sceptical of complex rather than elegant designs.

I know we're under time pressure but we run the risk of shipping something that is expensively broken here. As painful as it might seem, could it be worth revisiting the fundamental design of the system at this point?

Reply:
I'm open to other ideas. There is a refactor has been side-lined which is rather significant and streamlines a fair amount of the logic here. The line you're pointing to is a further constraint in place for the sake of protecting oracles. We could remove it, but I don't think the added constraint really makes the actual application more complicated.

Reply:
hey @samsondav, let's take general discussion on complexity to a meeting if need be, and keep this channel for specific feedback on the change